problems.rb

```diff
@@ -0,0 +1,6 @@
+def confusing_function_that_has_bugs(times)
```

Comment:
This function name could be a bit shorter, and it would help if it said a little bit about what the function does.